templates/_images/debian-11.yaml

```diff
@@ -1,3 +1,4 @@
+# DEPRECATED: Debian 11 (Bullseye) is deprecated. Use debian-12 (Bookworm) or debian-13 (Trixie) instead.
```

Comment:
As you intended to keep `templates/_images/debian-11.yaml`, you might have removed `templates/debian-11.yaml` by accident?